config/settings/base.py

```diff
@@ -611,3 +611,18 @@
 # Pool size for making parallel API requests
 MULTIPROCESSING_POOL_SIZE = env.int("MULTIPROCESSING_POOL_SIZE", 32)
 MULTIPROCESSING_TIMEOUT = env.int("MULTIPROCESSING_TIMEOUT", 128)
+
+# MATTERMOST
+# ------------------------------------------------------------------------------
+# URL of the Mattermost server.
+MATTERMOST_SERVER = env.str("MATTERMOST_SERVER")
+# Default is the https port but it can be configured
+MATTERMOST_PORT = env.int("MATTERMOST_PORT", 443)
+# Channel to ping
+MATTERMOST_CHANNEL = env.str("MATTERMOST_CHANNEL")
```

Comment:
Let's add a note (both here and in local and production `.envs/`) that the channel name should be lowercase and that the spaces should be replaced with minus signs (e.g. "Sprint Planning" -> "sprint-planning").